sections/clause_0_front_material.adoc

```diff
@@ -1,10 +1,27 @@
 [abstract]
 == Abstract
 
-This best practice document on UML to JSON encoding rules is established as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+This best practice document on UML to JSON encoding rules is initiated as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+
+Application schema in UML are used to model geospatial information as part of data specifications defining information and data content of domain related universe of discourse. In the geospatial domain the ISO 19103:2015 and ISO 19109:2015 UML profile is used, which therefore is also used in this document. Application schema operate at the conceptual level. At the implementation or data level JSON is one of the major data encodings used by current web applications. The UML to JSON encoding rules defined in the context of this document is a best practice. The aim of this document is to support the development of an international standard in this domain.
+
+To facilitate a proper JSON encoding an extension on the ISO 19103 and 19109 UML profile is proposed resulting in a UML-JSON encoding profile consisting of a list of extensions, typically of UML-stereotypes and related tagged values.
```

Comment:
Suggestion: "To facilitate a proper JSON encoding, an extension of the ISO 19103 and 19109 UML profiles is proposed, resulting in a UML-JSON encoding profile."

Reply:
ok

Reply:
done